qemu/tests/qemu_img_luks_key_management.py

```diff
@@ -0,0 +1,94 @@
+import re
+import json
+
+from virttest import data_dir
+from virttest import qemu_storage
+
+
+def run(test, params, env):
+    """
+    Luks key management by qemu-img amend command
+    1. Create a luks image
+    2. (1) Add a new password to a free keyslot (with_specified_index 1)
+       e.g. qemu-img amend --object secret,id=sec0,data=redhat
+            --object secret,id=sec1,data=amend
+            -o keyslot=1,state=active,new-secret=sec1
+            'json:{"file": {"driver": "file", "filename": "stg.luks"},
+            "driver": "luks", "key-secret": "sec0"}'
+       (2) check slots -> active is True
+    3. (1) Add a new password to a free keyslot (not setting keyslot index)
+       e.g qemu-img amend --object secret,id=sec0,data=redhat
+            --object secret,id=sec1,data=amend
+            -o state=active,new-secret=sec1
+            'json:{"file": {"driver": "file", "filename": "stg.luks"},
+            "driver": "luks", "key-secret": "sec0"}'
+       (2) check slots -> active is True
+    4. Negative test, overwrite active keyslot 0
+    5. Negative test, add a new password to invalid keyslot 8
+       It must be between 0 and 7
+    6. (1) Erase password from keyslot by giving a keyslot index
+       in this case, adding a password to keyslot 7 and then erase it
+       e.g. qemu-img amend --object secret,id=sec0,data=redhat
+            --object secret,id=sec1,data=amend
+            -o state=inactive,keyslot=7
+            'json:{"file": {"driver": "file", "filename": "base.luks"},
+            "driver": "luks", "key-secret": "sec0"}'
+       (2) check slots -> active is False
+    7. (1) Erase password from keyslot by giving the password
+       in this case, adding a password to keyslot 7 and then erase it
```

Comment:
adding a password to a free keyslot, and then erase it by password? or object id?

Reply:
Tested with two way (1) erase it with_specified_index (2) erase it with_specified_password, I will update the doc string. Thanks.